routers/web/repo/pull_review.go

```diff
@@ -228,14 +232,17 @@ func SubmitReview(ctx *context.Context) {
 	if err != nil {
 		if issues_model.IsContentEmptyErr(err) {
 			ctx.Flash.Error(ctx.Tr("repo.issues.review.content.empty"))
-			ctx.Redirect(fmt.Sprintf("%s/pulls/%d/files", ctx.Repo.RepoLink, issue.Index))
+			ctx.JSON(http.StatusOK, map[string]interface{}{
+				"redirectLink": fmt.Sprintf("%s/pulls/%d/files", ctx.Repo.RepoLink, issue.Index),
+			})
 		} else {
 			ctx.ServerError("SubmitReview", err)
 		}
 		return
 	}
-
-	ctx.Redirect(fmt.Sprintf("%s/pulls/%d#%s", ctx.Repo.RepoLink, issue.Index, comm.HashTag()))
+	ctx.JSON(http.StatusOK, map[string]interface{}{
+		"redirectLink": fmt.Sprintf("%s/pulls/%d#%s", ctx.Repo.RepoLink, issue.Index, comm.HashTag()),
```

Comment:
Looks like other places are using `"redirect":...` , maybe we should keep consistent?

Reply:
Updated in https://github.com/go-gitea/gitea/pull/25219/commits/f029cfbeb1af0af06aaa158d7a0432302c890911